scripts/benchmark.py

```diff
@@ -30,18 +28,17 @@ def main(
             log_file = open(log_path, "w")
             process = subprocess.Popen(
                 [
-                    "python",
-                    "-u",  # Unbuffered output
-                    "-m",
-                    "gpt_engineer.main",
-                    bench_folder,
-                    "--steps",
-                    "benchmark",
+                    "npx",
+                    "http-server",
```

Comment:
Don't actually use NPX, just run the python functions on separate ports. Also revert the comment removals